collector/cpu_freebsd.go

```diff
@@ -103,6 +103,8 @@ void freeCPUTimes(double *cpu_times) {
 */
 import "C"
 
+const MAXCPUTIMESLEN = C.MAXCPU * C.CPUSTATES;
```

Comment:
Everything which starts with an uppercase in go will be exported. The guide line is, don't export any types if not necessary. So I'd change this to something like `const maxCPUTimesLen = ...`.

https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps